app/controllers/api/v1/accounts/featured_tags_controller.rb

```diff
@@ -17,6 +21,6 @@ def set_account
   end
 
   def set_featured_tags
-    @featured_tags = @account.suspended? ? [] : @account.featured_tags
+    @featured_tags = @account.suspended? || disallow_unauthenticated_api_access? ? [] : @account.featured_tags
```

Comment:
```suggestion
@featured_tags = if @account.suspended? || disallow_unauthenticated_api_access?
[]
else
@account.featured_tags
end
```

Purely preference here, but I find ternary statements to be hard to read at the best of times. And a ternary with an or it it just seems ripe for misunderstanding.

Reply:
This is a great suggestion; I forgot that Ruby's `if`'s are expressions and not flow control.